src/main/java/domain/GameResultResponse.java

```diff
@@ -0,0 +1,16 @@
+package domain;
+
+import java.util.List;
+
+public class GameResultResponse {
+    private List<String[]> gameResult;
+    public GameResultResponse(List<String[]> gameResult) {
+        this.gameResult = gameResult;
+    }
+
+    public List<String[]> getGameResult() {
+        return gameResult;
+    }
+
+
```

Comment:
```suggestion
    private List<String[]> gameResult;
    
    public GameResultResponse(List<String[]> gameResult) {
        this.gameResult = gameResult;
    }

    public List<String[]> getGameResult() {
        return gameResult;
    }
```

공백 라인을 활용하면 가독성을 높이고 문맥을 구분지을수 있습니다 😄

Reply:
@sah3122 포맷팅을 적용하지 않았네요..체크해주셔서 감사합니다 :)